src/applications/disability-benefits/all-claims/components/ArrayField.jsx

```diff
@@ -0,0 +1,427 @@
+            {DescriptionField && (
+              <DescriptionField options={uiSchema['ui:options']} />
+            )}
+            {!textDescription && !DescriptionField && description}
+          </div>
+        )}
+
+        <div className="va-growable">
+          <Element name={`topOfTable_${idSchema.$id}`} />
+          {items.map((item, index) => {
+            // This is largely copied from the default ArrayField
+            const itemSchema = this.getItemSchema(index);
+            const itemIdPrefix = `${idSchema.$id}_${index}`;
+            const itemIdSchema = toIdSchema(
+              itemSchema,
+              itemIdPrefix,
+              definitions,
+            );
+            const isLast = items.length === index + 1;
+            const isEditing = this.state.editing[index];
+
+            if (isEditing) {
+              return (
+                <div key={index} className="va-growable-background">
+                  <Element name={`table_${itemIdPrefix}`} />
+                  <div className="row small-collapse">
+                    <div className="small-12 columns va-growable-expanded">
+                      {isLast &&
+                      items.length > 1 &&
+                      uiSchema['ui:options'].itemName ? (
+                        <h5>New {uiSchema['ui:options'].itemName}</h5>
+                      ) : null}
+                      <div className="input-section">
+                        <SchemaField
+                          key={index}
+                          schema={itemSchema}
+                          uiSchema={uiSchema.items}
+                          errorSchema={
+                            errorSchema ? errorSchema[index] : undefined
+                          }
+                          idSchema={itemIdSchema}
+                          formData={item}
+                          onChange={value => this.onItemChange(index, value)}
+                          onBlur={onBlur}
+                          registry={this.props.registry}
+                          required={false}
+                          disabled={disabled}
+                          readonly={readonly}
+                        />
+                      </div>
+                      <div className="row small-collapse">
+                        <div className="small-6 left columns">
+                          {!isLast && (
+                            <button
+                              className="float-left"
+                              onClick={() => this.handleUpdate(index)}
+                            >
+                              Update
+                            </button>
+                          )}
+                          {isLast && (
+                            <button
+                              className="float-left"
+                              disabled={!this.props.formData}
+                              onClick={this.handleSave}
+                            >
+                              Save
```

Comment:
This is the part you added, right?

Reply:
Yep. I added this button. Also added:
- handleSave action
- Modified handleRemove action